src/components/CreateEditNoteContainer/NoteForm/NoteForm.tsx

```diff
@@ -0,0 +1,64 @@
+import Button from '@mui/material/Button';
+import { useFormik } from 'formik';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router';
+import { useParams } from 'react-router-dom';
+import { addNote, editNote } from 'redux/notes-slice';
+import { AppStateType } from 'redux/redux-store';
+
+import styles from './NoteForm.module.scss';
+
+const NoteForm: React.FC<{ isEdit: boolean }> = ({ isEdit }) => {
+  const dispatch = useDispatch();
+  const { noteId } = useParams<{ noteId: string }>() || undefined;
+  // TODO set type for redux state
```

Comment:
also don't forget to remove this comment when `useAppSelector` will be added

Reply:
ready